Makefile

```diff
@@ -25,6 +25,10 @@ grow:
 	make create-random-branch
 	make merge-main
 
+lint:
+	@echo "\n––– Linting –––"
+	pre-commit run --all-files
```

Comment:
I've been using the makefile for experiments before rolling them out to our shared ops.

`tasks.py` is where our shared commands have been running until now, so I think this should be in `tasks.py` :-)

Reply:
This delineation is insufficiently documented, likely because it hasn't been relevant until now. We definitely want to remidy that.

Why keep invoke at all? Ovartaci.

Reply:
Tracked in #491.